### PR TITLE
indian-journal-of-orthopaedics

### PR DESCRIPTION
**Pull request by kbssrinivas**
There is no style named Indian Journal of Orthopaedics(IJO). This was made to make it easy for those authors submitting articles to IJO after going through requirements of the journal. Not sure if it resembles any other style.

Thanks,
Srinivas

### CODE REVIEW

File: Indian Journal of Orthopaedics [hunk 1]
```diff
@@ -0,0 +1,255 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" page-range-format="chicago" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Indian Journal of Orthopaedics</title>
+    <title-short>IJO</title-short>
+    <id>http://www.zotero.org/styles/indian-journal-of-orthopaedics</id>
+    <link href="http://www.zotero.org/styles/indian-journal-of-orthopaedics" rel="self"/>
+    <link href="http://www.ijoonline.com/contributors.asp" rel="documentation"/>
+    <author>
+      <name>Srinivas Kambhampati</name>
```
Comment (kbssrinivas) [inline Indian Journal of Orthopaedics]: Srinivas B S Kambhampati

**Comment by POBrien333** [comment]
Change filename to: indian-journal-of-orthopaedics.csl
Which style is this one based off. Please include that as outlined here: https://github.com/citation-style-language/styles/wiki/Style-Requirements

**Comment by kbssrinivas** [comment]
Please let me know How I can change file name? I am new to this website.
I am not sure which style this is based on. I had to significantly change it.  Will that be a problem?
Thanks,
Srinivas

**Comment by kbssrinivas** [comment]
<info>
     <link href="http://www.zotero.org/styles/clinical-gastroenterology-and-hepatology" rel="template"/>
   </info>

I think this is the style.

**Comment by kbssrinivas** [comment]
I think I found how to edit the file. I have given the closest template that I think I based it on.

**Comment by kbssrinivas** [comment]
Can I close this pull request? or will be it be closed by experts after reviewing?

**Comment by POBrien333** [comment]
Don't close it. (that does the opposite of what you want) The two people who run this repository and review styles for sign-off are busy academics with normal day jobs. Give it some more time. ;)

**Comment by kbssrinivas** [comment]
Thank you very much for your reply. I'm new to this. Will wait. I understand. They are doing a great job! I suppose it will take about a week to get an answer from them.

**Comment by rmzelle** [comment]
Thanks!

**Comment by kbssrinivas** [comment]
Should I delete the branch/ Fork? Can it be left alone?

**Comment by rmzelle** [comment]
@kbssrinivas, you can delete your branch/fork, or you can keep it. It doesn't really matter for us.

**Comment by kbssrinivas** [comment]
If I want to make a small change in the submitted file, how can I go about it? Or can I replace current file with another, better one with the same name?

**Comment by POBrien333** [comment]
You can search for the style now here and then click on the edit button on the top right corner, make the change and then click on Pull Request.
https://github.com/citation-style-language/styles

**Comment by kbssrinivas** [comment]
If I make the change and click the pull request,does the csl file present in the repository go offline? or does it continue to be available online until the change is approved when it changes to the new version?

**Comment by POBrien333** [comment]
The latter. A Pull Request is like a suggestion that is in the air. Only if it's accepted (and merged) the file will change for the users. Till then the current one will be available to everyone.

**Comment by POBrien333** [comment]
The latter. A Pull Request is like a suggestion that is in the air. Only if it's accepted (and merged) the file will change for the users. Till then the current one will be available to everyone.

**Pull request merged**